integration-test/src/main/java/com/sequenceiq/it/cloudbreak/util/azure/azurecloudblob/action/AzureCloudBlobClientActions.java

```diff
@@ -210,6 +215,9 @@ public String getLoggingUrl(String baseLocation, String clusterLogPath) {
                         containerName, keyPrefix, clusterLogPath));
             }
         } catch (Exception e) {
+            if (e.getCause() != null && e.getCause() instanceof TestFailException) {
```

Comment:
Not sure what this logic does actually, but wouldn't it be better to specify another catch clause instead of checking with instanceOf?